cmd/main.go

```diff
@@ -113,19 +113,19 @@ func readYAMLFile() error {
 	for i, entry := range baseline.Criteria {
 		// if entry in entryIDs
 		if slices.Contains(entryIDs, entry.ID) {
-			return fmt.Errorf("duplicate ID for criteria entry %d: %s", i, entry.ID)
+			return fmt.Errorf("duplicate ID for criterion entry %d: %s", i, entry.ID)
 		}
 		if entry.ID == "" {
-			return fmt.Errorf("missing ID for criteria entry %d: %s", i, entry.ID)
+			return fmt.Errorf("missing ID for criterion entry %d: %s", i, entry.ID)
```

Comment:
Never noticed this before, but if ID is "" we're going to have a hard time printing it. :-) I'm not actually sure what the behavior would be here. (Strictly speaking, this is probably worth fixing separately)